### PR TITLE
added remove feature

### PR DESCRIPTION
**Pull request by sln273**

### CODE REVIEW

File: storage.py [hunk 1]
```diff
@@ -12,8 +12,13 @@ def get(self, key):
         else:
             return None
 
-    def remove(self):
-        pass
+    def remove(self, key):
+        if key in self.data:
```
Comment (truepg) [inline storage.py:16]: Better to find key in self.data.keys(),  to my mind. It's an easier to read

**Review by truepg — CHANGES_REQUESTED** [review]
1. Find key in dict.keys()
2. Drop the second assert
3. Process the state when the key we wanna remove is not in our storage

**Review by truepg — APPROVED** [review]
good job

**Review by pavel1shatalov — APPROVED** [review]
Great! Ready to merge!

**Pull request merged**